### DISCUSSION TITLE
오늘도 이것저것

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#19] [General]
폴더 위치를 옮겨보았다.
![image](https://user-images.githubusercontent.com/62102830/192284282-7985967b-c7ad-41cf-a5c0-e754a722067a.png)
근데 이상하게 되버렸다 ㅋㅋㅋ

**Comment by Woon-2** [comment]
`Github Desktop`에서 `Local Repository` 위치를 갱신하지 않은 것 같아 보입니다.

`Locate`를 눌러 새롭게 갱신해주세요.

**Reply by Lcat12345** [reply]
![image](https://user-images.githubusercontent.com/62102830/192294634-90f17dc6-06a7-433e-ad6f-4363fef82c0f.png)
복구하였습니다.

**Comment by Lcat12345** [comment]
Locate...를 해도 그대롭니다.
repos 폴더(보통 visual studio에서 프로젝트 생성할 때의 경로)에 복사한후 삭제 했더니 이런 문제가 발생하더군요

**Reply by Woon-2** [reply]
복사하셨다 함은 무엇을 복사했다는 말이에요??
복사했다, 삭제했다의 대상이 무엇인지 궁금합니다.

**Reply by Lcat12345** [reply]
로컬 레퍼지토리 생성시 생기는 폴더를 복사후 삭제했습니다.(C₩document₩GitHub₩cpp-study)
->
(C₩user₩user₩source₩repos)

![Screenshot_20220927-121305_Chrome](https://user-images.githubusercontent.com/62102830/192423395-859e2272-b934-4053-910c-2ec04ad1dc0c.jpg)
 모바일이라 안보이는데 open in github desktop 으로 만들면 생기는 폴더입니다.
